packages/front-end/components/Features/ConditionDisplay.tsx

```diff
@@ -47,6 +47,18 @@ function operatorToText(operator: string): string {
       return `matches the pattern`;
     case "$notRegex":
       return `does not match the pattern`;
+    case "$veq":
+      return "version is equal to";
+    case "$vne":
+      return "version is not equal to";
+    case "$vlt":
+      return "version is less than";
+    case "$vlte":
+      return "version is less than or equal to";
+    case "$vgt":
+      return "version is greater than";
+    case "$vgte":
+      return "version is greater than or equal to";
```

Comment:
I had it that way to make it easier for the user to skim over and see that it was using the right comparison otherwise they'd have to go into edit mode then advanced mode to view the syntax to ensure it's done correctly. I think using the same copy for both makes it harder to skim, but regardless, I've made the change based on your request: 47e589ccbf84a23ca732ba80d27dcc4f9e201c8a